build.sbt

```diff
@@ -398,6 +398,14 @@ lazy val scalalib =
       compile in Compile := (compile in Compile)
         .dependsOn(assembleScalaLibrary)
         .value,
+      // Don't include classfiles for scalalib in the packaged jar.
+      mappings in packageBin := {
```

Comment:
Builds fails at the moment. It seems that `in Compile` is missing here and on the next line, [similarly to the mapping for javalib](https://github.com/carrot-garden/scala_scala-native/blob/81d76e2c1fdb01698749bf9cd2445a567660bdd5/build.sbt#L321).

Reply:
changed to `mappings in packageBin in Compile`